### DISCUSSION TITLE
new plugin suggestion: Local Storage for IOS & Android

### DISCUSSION THREAD

**Discussion by anisg** [tauri-apps/tauri#14539] [Plugin Requests]
Hello team,

I’m developing an iOS app with Tauri and needed to use Swift UserDefaults for storage. 

I had to create a custom Tauri v2 plugin that contains some Swift code just for this, which was a bit hard.

It would be great if Tauri had a local storage plugin for IOS & Android, that would make thing easier. 

Best,
Anis

**Comment by FabianLars** [comment]
Could _probablyyy_ be integrated into the existing store plugin (which uses plain files) but added the `new plugin` label for now. Thanks for the request!

p.s. any chance you could share a bit of the code you ended up using (or any kind of info) for anyone who would like to take on this task?
